pages/clubs/StepTwo.jsx

```diff
@@ -0,0 +1,357 @@
+          </div>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <label>{ labels[this.state.intent].audienceType }</label>
+        <input className={this.error('audienceType')} type="text" value={this.state.audienceType} onChange={this.updateAudienceType} placeholder="Students, professionals, community leaders, etc..."/>
+      </fieldset>
+
+      <fieldset>
+        <label>{ labels[this.state.intent].meetingSubjects }</label>
+        <input className={this.error('meetingSubjects')} type="text" value={this.state.meetingSubjects} onChange={this.updateMeetingSubjects} placeholder="Web literacy, 21st century skills, online privacy, social media, etc..."/>
+      </fieldset>
+
+      <fieldset>
+        <label>Affiliated institution or oganization</label> (optional)
+        <input type="text" value={this.state.affiliation} onChange={this.updateAffiliation} placeholder="Name of the school, library, organization, etc..."/>
+      </fieldset>
+      
+      {
+        this.state.intent === 'integrate' ?
+          <fieldset>
+            <label>Your current club website</label> (optional)
+            <input className={this.error('website')} type="text" value={this.state.website} onChange={this.updateWebsite} placeholder="https://your.clubsite.com"/>
+          </fieldset>
+          :
+          null
+      }
+
+      <fieldset>
+        <div className="pledge"><input className={this.error('pledgeAgreement')} type="checkbox" checked={this.state.pledgeAgreement} onChange={this.updatePledgeAgreement}/> I agree to the <a target="_blank" href="http://soapdog.github.io/your-first-month-as-a-club-captain-guide/#mozilla-club-captain-pledge">Mozilla Club Captain Pledge</a>.</div>
+      </fieldset>
+
+      { this.renderValidationErrors() }
+    </div>;
+  },
+
+  updateIntent: function(evt) { this.setStateAsChange({ intent: evt.target.value, errors: []}, true); },
+  updateClubName: function(evt) { this.setStateAsChange({ clubName: evt.target.value }); },
+  updateDescription: function(evt) { this.setStateAsChange({ description: evt.target.value }); },
+  updateMeetingVenue: function(evt) { this.setStateAsChange({ meetingVenue: evt.target.value }); },
+
+  updateFrequency: function(evt) { this.setStateAsChange({ frequency: evt.target.value }); },
+  updateFrequencyOther: function(evt) { this.setStateAsChange({ frequencyOther: evt.target.value }); },
+
+  updateAgeRange: function(evt) {
+    var val = evt.target.value;
+    var ar = this.state.ageRange;
+    var pos = ar.indexOf(val);
+    if (pos > -1) { ar.splice(pos,1); } else { ar.push(val); }
+    this.setStateAsChange({ ageRange: ar });
+  },
+  updateAgeRangeOther: function(evt) { this.setStateAsChange({ ageRangeOther: evt.target.value }); },
+
+  updateClubSize: function(evt) { this.setStateAsChange({ clubSize: evt.target.value }); },
```

Comment:
This and above seem WET. Any reason the inputs can't share an event that gets the field name automatically?